internal/packagemanager/packagemanager.go

```diff
@@ -47,6 +53,41 @@ const (
 	ssmPkgName      = "amazon-ssm-agent"
 )
 
+// Default containerd repository content
+// TODO: Change to include download from closest region
+const containerdRepoContent = `deb http://us-west-2.ec2.archive.ubuntu.com/ubuntu jammy-updates main
```

Comment:
why do we need to add these? are they arent already configured on a default Ubuntu install?